src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -92,6 +92,7 @@ def create_webapp(cmd, resource_group_name, name, plan, runtime=None, startup_fi
                   using_webapp_up=False, language=None, assign_identities=None,
                   role='Contributor', scope=None, vnet=None, subnet=None, https_only=False):
     from azure.mgmt.web.models import Site
+    from azure.core.exceptions import ResourceNotFoundError as _ResourceNotFoundError
```

Comment:
Out of curiosity, I'd like to ask why you want to alias `ResourceNotFoundError` as `_ResourceNotFoundError`?

Reply:
The ResourceNotFoundError here is from azure.core.exceptions instead of from azure.cli.core.azclierror, and in the file we are already using ResourceNotFoundError from azure.cli.core.azclierror.

Reply:
Got it~ Thanks~